comment_gen.go

```diff
@@ -23,46 +27,6 @@ func (c Comment) Child(ChildIdx uint16) Comment {
 	return Comment{C.clang_Comment_getChild(c.c, C.uint(ChildIdx))}
 }
 
-// A \c CXComment_Paragraph node is considered whitespace if it contains only \c CXComment_Text nodes that are empty or whitespace. Other AST nodes (except \c CXComment_Paragraph and \c CXComment_Text) are never considered whitespace. \returns non-zero if \c Comment is whitespace.
-func (c Comment) IsWhitespace() bool {
```

Comment:
There is something wrong. There should never be a removal of existing generations, unless we generated them wrong in the first place.